src/com/ecfeed/core/generators/algorithms/NwiseScoreEvaluator_Rep.java

```diff
@@ -0,0 +1,151 @@
+package com.ecfeed.core.generators.algorithms;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import com.ecfeed.core.generators.api.GeneratorException;
+import com.ecfeed.core.generators.api.IConstraintEvaluator;
+import com.ecfeed.core.utils.EvaluationResult;
+
+public class NwiseScoreEvaluator_Rep<E> implements IScoreEvaluator<E> {
```

Comment:
What the name NwiseScoreEvaluator_Rep means? What is _Rep for?

Reply:
Just used to distinguish the code with the previous version. I have removed "-Rep" now as it seems we both have agreed on the code :)